tests/cupy_tests/manipulation_tests/test_add_remove.py

```diff
@@ -29,3 +29,97 @@ def test_unique_inverse(self, xp, dtype):
     def test_unique_counts(self, xp, dtype):
         a = testing.shaped_random((100, 100), xp, dtype)
         return xp.unique(a, return_counts=True)[1]
+
+
+@testing.gpu
+class TestTrim_zeros(unittest.TestCase):
```

Comment:
How about using `testing.parameterize` to parameterize the input arrays?

Reply:
Please add a test for zero-size input and non-1dim input.

Reply:
ndim input is not handled in numpy. Should I raise some error, flatten, or leave without handling ? The current version doesn't change non-1dim input.

Reply:
Please check both NumPy and CuPy raise the error of the same type.

Reply:
Please add a test for 0-dim input.